src/Symfony/Component/Notifier/Bridge/Smsbox/Enum/Mode.php

```diff
@@ -15,5 +15,5 @@ enum Mode: string
 {
     case Standard = 'Standard';
     case Expert = 'Expert';
-    case Response = 'Reponse';
+    case Response = 'Reponse'; # This is not a typo, it's the French word for "response" which is being used by the API (see https://en.smsbox.net/docs/doc-API-SMSBOX-1.1-EN.pdf)
```

Comment:
Yes, I would keep it simpler:

```suggestion
    case Response = 'Reponse'; // not a typo, see https://en.smsbox.net/docs/doc-API-SMSBOX-1.1-EN.pdf
```